OpenRA.Mods.Common/Orders/BeaconOrderGenerator.cs

```diff
@@ -28,9 +28,12 @@ protected override IEnumerable<Order> OrderInner(World world, CPos cell, int2 wo
 		protected override IEnumerable<IRenderable> Render(WorldRenderer wr, World world) { yield break; }
 		protected override IEnumerable<IRenderable> RenderAboveShroud(WorldRenderer wr, World world) { yield break; }
 		protected override IEnumerable<IRenderable> RenderAnnotations(WorldRenderer wr, World world) { yield break; }
+
+		public string BeaconCursor;
```

Comment:
Move it above, before `OrderInner()`: fields are (usually) usually ordered before methods.